INSTALL.md

```diff
@@ -40,6 +40,13 @@ $> git clone https://github.com/savoirfairelinux/sous-chef
 $> docker-compose build
 $> docker-compose up
```

Comment:
After installing Docker Beta for Mac, I was able to make this work. The only change was in docker-compose.yml. Are you ok with this?

```
(snip)
  web:
    restart: always
    build: .
    depends_on:
      - db

  nginx:
    restart: always
    build: tools/nginx
    command: /bin/bash -c "nginx"
    ports:
      - "80:80"
    volumes:
      - ./src/sous-chef/static:/usr/share/nginx/html
    depends_on:
      - web
```